src/components/Jobs/JobDetails/index.tsx

```diff
@@ -350,8 +350,15 @@ const JobDetailsEdit: FC<IJobDetailsEdit> = ({
     setEditorVal(jobEntryData.description);
   }, []);
 
+  const getStatus = () => {
+    if (jobEntryData.jobStatus === 'inactive') {
+      return false;
+    }
+    return true;
```

Comment:
`return jobEntryData.jobStatus === 'active'`
A simpler way to write the condition